generic/tests/cfg/mac_change.cfg

```diff
@@ -13,7 +13,7 @@
         - @up_change:
             virtio_net:
                 no RHEL.5, RHEL.6.0, RHEL.6.1, RHEL.6.2, RHEL.6.3
-                no SLES.10.4 SLES.11.0 SLES.11.1 SLES.11.2 SLES.11.3
+                no SLES.10.4 SLES.11.0 SLES.11.1 SLES.11.2 SLES.11.3 SLES.11.4
```

Comment:
We can't do that.

As SLES.12 support it and maybe other enterprises support it too.

Thanks.